string robot_desc
string serial_num
string ip_address
string firmware_ver
string max_payload
string hydraulic_oil_temperature
string hydraulic_oil_tank_fill_level
string hydraulic_oil_pressure
